facebook/__init__.py

```diff
@@ -163,10 +163,44 @@ def put_object(self, parent_object, connection_name, **data):
 
         """
         assert self.access_token, "Write operations require an access token"
+
         return self.request(
             "{0}/{1}/{2}".format(self.version, parent_object, connection_name),
             post_args=data,
             method="POST")
+                        
+        
+    def get_access_token_for_page(self, profile_id="me", page_id=None):
+        """
+        Retrive access token of user as admin of Page
+        """
+        if not page_id:
+            return None
+        response = self.request(profile_id + "/accounts")
+        for token in response['data']:
+            if str(page_id) in token['id']:
+                return token['access_token']
+        return None
+
+    def put_wall_post(self, message, attachment={}, profile_id="me"):
```

Comment:
The `put_wall_post` method was removed in 88dbc2f33ec566872188fbad38d8677e4489f5ec; I think the last merge comment re-added it by mistake.